src/configs/recommended.ts

```diff
@@ -17,14 +17,9 @@
 
 export const rules = {
     "adjacent-overload-signatures": true,
-    align: {
-        options: ["parameters", "statements"],
-    },
     "array-type": {
-        options: ["array-simple"],
+        options: ["array"],
```

Comment:
This is the only non-formatting change here. The default's inconsistency has been bugging me for a while... I can remove or revert this if you'd prefer?

Reply:
Fair enough, the inconsistency is a little confusing...